core/process.py

```
@@ -39,6 +39,36 @@ def sweep_stale_persona_payloads(payload_dir: str, ttl_seconds: int = PERSONA_PA
+    """
+    script_path = os.path.join(
+        os.path.dirname(__file__), "..", "scripts", "meetingbaas.py"
+    )
+    try:
+        process = subprocess.Popen(
+            [sys.executable, script_path, "--warmup"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            text=True,
+        )
+    except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)